expression/expr_to_pb.go

```diff
@@ -90,14 +90,14 @@ func NewPBConverter(client kv.Client, sc *stmtctx.StatementContext) PbConverter
 }
 
 // ExprToPB converts Expression to TiPB.
-func (pc PbConverter) ExprToPB(expr Expression) *tipb.Expr {
+func (pc PbConverter) ExprToPB(sc *stmtctx.StatementContext, expr Expression) *tipb.Expr {
```

Comment:
`PbConverter` already has a `StatementContext` field, why do we add a new `sc` into its arguments?

Reply:
addressed